pipeline/load.py

```diff
@@ -1,28 +1,53 @@
 """Handles all load functions that load data and information into the database"""
 
 from os import environ
-
+import pandas as pd
 from dotenv import load_dotenv
-from psycopg2 import connect
-from psycopg2.extensions import connection, cursor
+from psycopg2 import connect, OperationalError
+from psycopg2.extensions import connection
 
+TRANSFORMED_DATA = ""
 
-def get_db_connection(config) -> connection:
+def get_db_connection() -> connection:
     """Returns connection to the rds database"""
-    return connect(host=config["DB_HOST"],
-                   user=config["DB_USER"],
-                   password=config["DB_PASSWORD"],
-                   port=config["DB_PORT"],
-                   dbname=config["DB_NAME"])
+    load_dotenv()
+    try:
+        return connect(host=environ["DB_HOST"],
+                   user=environ["DB_USER"],
+                   password=environ["DB_PASSWORD"],
+                   port=environ["DB_PORT"],
+                   dbname=environ["DB_NAME"])
+    except OperationalError:
+        print("Failed to connect to database.")
+        raise OperationalError()
```

Comment:
why catch the error to raise it again?

Reply:
i needed to break the script somehow

Reply:
very open to suggestions

Reply:
if you let it raise the error without catching it will it not break?

Reply:
but I want to make the impression that I expected this error

Reply:
fixed